docs/index.rst

```diff
@@ -5,9 +5,10 @@
     :caption: Table of Contents:
     :hidden:
 
-    generic/theory
     generic/install
     python/index
+    python/tutorial
     matlab/api
-    generic/references
+    matlab/tutorial
```

Comment:
matlab/tutorial appears to be a copy of matlab/api i.e. we've got the same page at the top-level now. This should probably be merged into matlab/index which can serve as a top-level for MATLAB-specifc documentation